public/app/core/logs_model.ts

```diff
@@ -136,30 +136,29 @@ export function makeSeriesForLogs(rows: LogRowModel[], intervalMs: number, timeZ
   }
 
   return seriesList.map((series, i) => {
-    series.datapoints.sort((a: number[], b: number[]) => {
-      return a[1] - b[1];
-    });
+    series.datapoints.sort((a: number[], b: number[]) => a[1] - b[1]);
 
     // EEEP: converts GraphSeriesXY to DataFrame and back again!
     const data = toDataFrame(series);
-    const points = getFlotPairs({
-      xField: data.fields[1],
-      yField: data.fields[0],
-      nullValueMode: NullValueMode.Null,
-    });
 
-    const timeField = data.fields[1];
+    const timeField = data.fields.find(field => field.type === FieldType.time);
```

Comment:
Nitpick: We could use FieldCache.getFirstFieldOfType here.

Reply:
oh nice, didn't know about this